tests/zero_code_change/test_tensorflow2_integration.py

```diff
@@ -134,7 +143,7 @@ def helper_test_keras_v2_json_config(
         trial = smd.create_trial(path=sim.out_dir)
         assert len(trial.steps()) > 0, "Nothing saved at any step."
         assert len(trial.tensor_names()) > 0, "Tensors were not saved."
-        if not eager_mode:
+        if not eager_mode and is_tf_2_3() is False:
```

Comment:
what about when its not eager and its TF 2.3?

Reply:
If `is_tf_2_3() == True`  then always skip the test.
(not eager and its TF 2.3) will evaluate to (True and False) which is False.